packages/client/hmi-client/src/page/project/components/tera-upload-resources-modal.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+	<Teleport to="body">
```

Comment:
In most other places, the <Teleport> semantic is left to the caller of the modal-component - should we stick with this convention or is there a particular reason for breaking tradition here?

Reply:
Yes. For the user of the modal componant, having to do`<Teleport><modal/></Teleport>` seems less unintuitive and more cumbersome than simply being able to use `<modal/>`

Reply:
I.e. leaving <Teleport> as the responsibility of the caller of the modal component was a mistake I made when I created `tera-modal`

Reply:
This is a valid reason, but this is not a good time to both add new features and refactor simultaneously. 
- It introduces inconsistencies and confusions into the code base, it is unclear which style is the correct approach, and it is unclear whether a clean up effort will be initiated.
- There is no communication about this change or the follow up steps, which would have saved some back and forth communications. For what it is worth we simply can't tell if this was intentional, or if it was just some copy pasta mistake.
- By your argument, `<Teleport>` should be inside `tera-modal` component, not here.